openc3/lib/openc3/packets/parsers/xtce_parser.rb

```diff
@@ -699,6 +699,8 @@ def set_min_max_default(item, type, data_type)
               # Strip quotes from strings
               if type.initialValue[0] == '"' && type.initialValue[-1] == '"'
                 item.default = type.initialValue[1..-2]
+              else
+                item.default = type.initialValue
```

Comment:
Are quote optional with XTCE?

Reply:
Apparently. Someone ran into this case in COSMOS 4.